src/components/text-style-button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface textButtonInterface {
+    text: string
+    onClicked: () => void
+}
+
+
+const ButtonStyle = styled.div`
+    color: #BDBDBD;
+    font-family: Montserrat, Arial;
+    font-size: 14px;
+    cursor: pointer;
+    &:hover {
+        text-decoration: underline;
+    }
+`
+
+function TextButton(props: textButtonInterface) {
```

Comment:
So you can do this cool thing called object destructuring, where instead of assigning the whole object to a variable called props, it takes out the keys of the object that you specify and they are assigned to variables per their name (ie instead of `pops.onClicked` you would just specify `onClicked` because of the object destructuring.
```suggestion
function TextButton({ text, onClicked }: textButtonInterface) {
```